### DISCUSSION TITLE
fail to join slack with error

### DISCUSSION THREAD

**Discussion by nobody4t** [milvus-io/milvus#28001] [Show and tell]
I tried to join the slack channel. But I got the error message when to join with google account..

```
The email address must match one of the domains listed below. Please try another email.
```

**Comment by yhmo** [comment]
We’re moving the Milvus community from Slack to Discord so we can do more live sessions, contests and more.
Join us here: https://discord.gg/4UhDeYewVk

Our last office hours via Zoom/Slack will be on 10/31. Going forward we’ll host all live sessions in the Discord. The Slack channel will still be available until November 15, 2023 and will no longer accept messages to shared channels past that date.
